### PR TITLE
Midnight sun spoilers 2022 07 22

### PR DESCRIPTION
**Pull request by plural**

### CODE REVIEW

**Review by NoahTheDuke — APPROVED** [review]

**Comment by plural** [comment]
hahahahahahaha.  You set an unrealistic PR review standard.  That was SECONDS.

**Comment by distributive** [comment]
That's not fair, this dropped in the middle of the night for me!

File: pack/ms.json [hunk 1]
```diff
@@ -735,7 +776,7 @@
     "code": "33039",
     "deck_limit": 3,
     "faction_code": "jinteki",
-    "flavor": "Some tests require <strong>specific</strong> conditions.",
+    "flavor": "Some tests require specific conditions.",
```
Comment (distributive) [inline pack/ms.json:779]: Am I wrong in thinking this flavour text is bolded, or is this an artefact of the source sheet missing that detail?

Comment (plural) [reply pack/ms.json:779]: no, it needs the bold.  I'll fix it.

**Comment by NoahTheDuke** [comment]
Are these the final cards needed to have the complete set?

**Comment by plural** [comment]
Yep! I'll push this when I get home this morning

**Pull request merged**